src/js/menu/menu-item.js

```diff
@@ -68,6 +69,26 @@ class MenuItem extends ClickableComponent {
     }, props), attrs);
   }
 
+  /**
+   * Ignore keys which are used by the menu, but pass any other ones up. See
+   * {@link ClickableComponent#handleKeyPress} for instances where this is called.
+   *
+   * @param {EventTarget~Event} event
+   *        The `keydown` event that caused this function to be called.
+   *
+   * @listens keydown
+   */
+  handleKeyPress(event) {
+    const keyCode = keyboardKey.getCode(event);
+
+    if (keyCode !== keyboardKey.Escape && keyCode !== keyboardKey.Tab &&
```

Comment:
These are all keycodes that `menu.js` uses? Is there a way to centralize this so that if we modify menu.js we don't break things by forgetting to add stuff in here?

Reply:
Done in the latest change